app/storybook/documentation/ui-kit/ChartJS.mdx

```diff
@@ -0,0 +1,30 @@
+import { Meta } from '@storybook/blocks'
+import { SourceCode } from '../../.storybook/blocks/SourceCode'
+import * as ThemeProviderStories from '../../../../packages/ui-kit/src/components/ThemeProvider/ThemeProvider.stories'
+import * as TimeSeriesStories from '../../../../packages/ui-kit/src/components/TimeSeries/TimeSeries.stories'
+
+<Meta title="Customization/Chart configuration" />
+
+# Chart Configuration
+
+We built the UI Kit on top of all the power and efficiency of Chart.js.
+
+## Global Chart.js configuration
+
+You can extend or override global Chart.js settings by providing a `globalChartConfigProps` function to the
+`ThemeProvider` component. This function accepts the [config object](https://www.chartjs.org/docs/latest/configuration/)
+from Chart.js as an argument and should return the modified config object with your configurations applied.
+
+Note that the settings will be applied globally to Chart.js throughout the app, affecting all chart instances, not only
+those within the `ThemeProvider`.
```

Comment:
Since we are making a breaking change anyway, yes, I think it makes sense to get rid of the global. It wouldn't be hard to reintroduce some version of it later, if really necessary.